test/models/user_test.rb

```diff
@@ -3,7 +3,49 @@
 require 'test_helper'
 
 class UserTest < ActiveSupport::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
+  test '#following?' do
+    alice = users(:alice)
+    bob = users(:bob)
+
+    assert_not alice.following?(bob)
+    Relationship.create(follower_id: alice.id, following_id: bob.id)
```

Comment:
```suggestion
    Relationship.create(follower: alice, following: bob)
```

でも動くかも？（未検証）

あと、createに失敗したらそこでテストが落ちるように`create!`を使うとさらに良いです。

Reply:
> でも動くかも？（未検証）

上の記述に修正して、動きました！
（冗長な書き方になってました。。）

> createに失敗したらそこでテストが落ちるようにcreate!を使うとさらに良いです。

create!に修正しました。例外を発生させる意図ですね😊

参考：[【Rails/ActiveRecord】createとcreate\!の違いについて \- Qiita](https://qiita.com/saika_0/items/20adc17112d6d0bcf9d5)

あと、コメントのSuggested changeってどうやっているのだろうと思って調べたら、コメント時の左から一番目のアイコンをクリックすれば良いのですねー。
参考：[君は GitHub の Suggested change を知っているか？ \- BASEプロダクトチームブログ](https://devblog.thebase.in/entry/2018/12/02/130657)